rubygem-parse-cron/rubygem-parse-cron.spec

```diff
@@ -0,0 +1,76 @@
+%{?scl:%scl_package rubygem-%{gem_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global gem_name parse-cron
+
+Summary: Cron expression parser
+Name: %{?scl_prefix}rubygem-%{gem_name}
+Version: 0.1.4
+Release: 1%{?dist}
+Epoch: 1
+Group: Development/Languages
+
+License: MIT
+URL: https://github.com/siebertm/parse-cron
+Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
+Requires: %{?scl_prefix_ruby}ruby(rubygems)
+Requires: %{?scl_prefix_ruby}ruby
+
+%if "%{?scl_ruby}" == "ruby193" || (0%{?el6} && 0%{!?scl:1})
```

Comment:
I could see it maybe being used on the smart proxy at some point in the future, still want it removed for now?